### PR TITLE
Fix DeliveryList and DeliveryView Use Case

### LINKED ISSUE #513: Update DG - Delivery List & View Use Cases

### PR DESCRIPTION
**Pull request by juliusgambe**
- Change point form of `List Delivery`
- Change error format of `View Delivery` (Adapted from @B-enguin)

Closes #513

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -1479,80 +1479,113 @@ otherwise)
...
 
    Use Case Ends.
 
 **Extensions:**
 
-* 1a. User specifies status field in command.
-    * 1a1. DMS display a list of deliveries filtered by the specified status.
+- 1a. User specifies status field in command.
+
+    - 1a1. DMS display a list of deliveries filtered with the specified status.
```
Comment (zhonghan721) [inline docs/DeveloperGuide.md]: Would it be `displays`?

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -1479,80 +1479,113 @@ otherwise)
...
+
+    - 1c1. DMS displays a list of deliveries filtered by the specified expected delivery date.
 
       Use Case Ends.
 
-* 1d. User specifies both status and customer fields.
-    * 1c1. DMS displays a list of deliveries filtered by the specified status and customer.
+- 1d. User specifies expected delivery date field as "TODAY" in command.
+
+    - 1d1. DMS displays a list of deliveries filtered by the expected delivery date that is today's date.
```
Comment (zhonghan721) [inline docs/DeveloperGuide.md]: Not sure if this would be considered giving too much details.
Similarly for 1g, 1j, 1k

**Review by zhonghan721 — COMMENTED** [review]
Some general comments

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (juliusgambe) [reply docs/DeveloperGuide.md]: Like "TODAY" is it?

Comment (zhonghan721) [reply docs/DeveloperGuide.md]: Yep

Comment (juliusgambe) [reply docs/DeveloperGuide.md]: Hmm but because our `--date` accepts either `TODAY` or an actual date, it could be quite different?

**Review by jianyangg — APPROVED** [review]
LGTM

**Pull request merged**